src/app/services/signing/signing.service.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { inject, TestBed } from '@angular/core/testing';
+import { Http } from '@angular/http';
+import { HttpMock } from './http.mock';
+import { SigningService } from './signing.service';
+
+describe('SigningService', () => {
+  let service: SigningService;
+  let http: HttpMock;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
```

Comment:
as already discussed: why so complicated? you can treat the `SigningService` as a normal object and construct it yourself, especially when you mock everything else away :) no need to use `TestBed` here.